templates/template/README.md

```diff
@@ -79,24 +78,103 @@ Before submitting any PRs, a Github Workflow will check to see if your package d
 
 {{- end }}
 
+{{- if (eq .Template "source") }}
+### Making Changes
+
+As a developer making changes to a particular package, you will usually follow the following steps:
+
+#### If this is the first time you are adding a package:
+
+```shell
+PACKAGE=<packageName>
+mkdir -p packages/${PACKAGE}
+touch packages/${PACKAGE}/package.yaml
+```
+
+See `packages/README.md` to configure the `packages/${PACKAGE}/package.yaml` file based on the Package that you are planning to add. To make changes, see the steps listed below.
+
+#### If the package already exists
+
+If you are working with a single Package, set `export PACKAGE=<packageName>` to inform the scripts that you only want to make changes to a particular package. This will prevent the scripts from running commands on every package in this repository.
+
+You'll also want to update the `packageVersion` and `releaseCandidateVersion` located in `packages/${PACKAGE}/package.yaml`. See the section below for how to update this field.
```

Comment:
nit: split sentences into multiple lines

Reply:
addressed in latest, thanks for catching!